pkg/render/render.go

```diff
@@ -103,6 +103,14 @@ func nindentPrefix(spaces int, prefix, v string) string {
 	return strings.Replace(nindent(spaces, prefix+v), " ", "", len(prefix))
 }
 
+// imagePath returns contaimer image path, supporting sha256 format
+func imagePath(repository, image, version string) string {
```

Comment:
Why do we need it as a template function instead of generating it in the code and pass the value into template? IMO, it makes templates more complicated whithout additional benefits

Reply:
My thoughts that this way will require less changes.
In code, we will need for each state:
- add a new field 'imagePath' to the `ManifestRenderData`
- generate the value and add it in the `ManifestRenderData`
- update the manifests to use the new 'imagePath'

I am OK to do the change if you prefer that way

Reply:
Sounds good to me to keep it as proposed now